code/modules/hydroponics/grown/root.dm

```diff
@@ -34,7 +34,7 @@
 /obj/item/reagent_containers/food/snacks/grown/carrot/attackby(obj/item/I, mob/user, params)
 	if(is_sharp(I))
 		to_chat(user, "<span class='notice'>You sharpen the carrot into a shiv with [I].</span>")
-		var/obj/item/kitchen/knife/carrotshiv/Shiv = new /obj/item/kitchen/knife/carrotshiv
+		var/obj/item/kitchen/knife/shiv/carrot/Shiv = new /obj/item/kitchen/knife/shiv/carrot
```

Comment:
```suggestion
		var/obj/item/kitchen/knife/shiv/carrot/Shiv = new
```
`new` takes the type of the var you're assigning to if you omit specifying one

Reply:
Thank you so much for catching that! It was an obvious oversight on my part. I went back to look at the file to understand it better, add your edit, and rebuild again just to make 100% sure nothing changed.